src/bindings/python/tests/test_runtime/test_infer_request.py

```diff
@@ -678,15 +680,14 @@ def test_query_state_write_buffer(device, input_shape, data_type, mode):
     from openvino.runtime.utils.types import get_dtype
 
     model = create_model_with_memory(input_shape, data_type)
+    model.validate_nodes_and_infer_types()
```

Comment:
My only concern: why is this call needed here? Is it going to be needed for all cases when model has `read_value` as operator?

Reply:
This method checks the model consistency and it throwed an exception without the fixes.
So I've just added it be sure that it passes.
Maybe we need to add this call for other tests as well.